core/src/main/java/io/cloudevents/core/format/ContentType.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.cloudevents.core.format;
+
+import io.cloudevents.CloudEvent;
+import io.cloudevents.CloudEventData;
+import io.cloudevents.rw.CloudEventDataMapper;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * <p>A construct that aggregates a two-part identifier of file formats and format contents transmitted on the Internet.
+ *
+ * <p>The two parts of a {@code ContentType} are its <em>type</em> and a <em>subtype</em>; separated by a forward slash ({@code /}).
+ *
+ * @see io.cloudevents.core.format.EventFormat
+ */
+@ParametersAreNonnullByDefault
+public enum ContentType {
+
+    /**
+     * Content type associated with the CSV event format
+     */
+    CSV("application/cloudevents+csv"),
+    /**
+     * Content type associated with the JSON event format
+     */
+    JSON("application/cloudevents+json"),
+    /**
+     * The content type for transports sending cloudevents in the protocol buffer format.
+     */
+    PROTO("application/cloudevents+protobuf"),
+    /**
+     * The content type to set for the "datacontenttype" attribute if the data is stored in protocol buffer format.>
+     * Note that if this content type is used, the stored data must be wrapped in {@link com.google.protobuf.Any} as specified by the protobuf format spec.>
+     */
+    PROTO_DATA("application/protobuf"),
```

Comment:
I'm not sure we need this as it's not a CloudEvent format media type.

We don't enumerate application/json for example.